webui/react/src/hooks/usePolling.ts

```diff
@@ -5,14 +5,18 @@ type PollingFn = (() => Promise<void>) | (() => void);
 interface PollingHooks {
   isPolling: boolean;
   startPolling: () => void;
-  stopPolling: () => void;
+  stopPolling: (options?: StopOptions) => void;
 }
 
 interface PollingOptions {
   interval?: number;
   runImmediately?: boolean;
 }
 
+interface StopOptions {
+  terminateGracefully?: boolean;
```

Comment:
nit: could you add a comment as to what this option would/should do so it's externally more easily understandable?